testsuite/features/secondary/srv_power_management_redfish.feature

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Power management
+
+  Scenario: Fake a Redfish host
+    Given the server starts mocking a Redfish host
+
+  Scenario: Save power management values for Redfish
+    Given I am on the Systems overview page of this "sle_minion"
+    And I follow "Provisioning" in the content area
+    And I follow "Power Management" in the content area
+    When I enter "server" as the redfish server address
+    And I enter "ipmiusr" as "powerUsername"
+    And I enter "test" as "powerPassword"
+    And I select "Redfish" from "powerType"
+    And I click on "Save"
+    Then I should see a "Power settings saved" text
+    And the cobbler report contains ":8443" for system "sle_minion"
+    And the cobbler report contains "Power Management Username      : ipmiusr" for system "sle_minion"
+    And the cobbler report contains "Power Management Password      : test" for system "sle_minion"
+    And the cobbler report contains "Power Management Type          : redfish" for system "sle_minion"
+
+  Scenario: Test Redfish functions
+    Given I am on the Systems overview page of this "sle_minion"
+    And I follow "Provisioning" in the content area
+    And I follow "Power Management" in the content area
+    Then I click on "Power On"
+    And I click on "Get status"
+    And I should see the power is "On"
+    And I click on "Power Off"
+    And I should see a "system has been powered off" text
+    And I wait for "20" seconds
+    And I should see the power is "Unknown"
```

Comment:
Please just add a timeout of 20 seconds inside the step definition of `I should see the power is "Unknown"`, so we can get rid of steps like `I wait for x seconds`

Reply:
I think the wait is useless. Maybe a artifact from the times when we used a real IPMI system. I will remove it